packages/forms/src/styled/select/StyledSelect.ts

```diff
@@ -18,9 +18,11 @@ const colorStyles = (props: IStyledTextInputProps & ThemeProps<DefaultTheme>) =>
 
   /* prettier-ignore */
   return css`
+    /* stylelint-disable-next-line no-duplicate-selectors */
     &:hover + ${StyledTextMediaFigure},
     &:focus + ${StyledTextMediaFigure},
-    &[data-garden-focus-visible='true'] + ${StyledTextMediaFigure} {
+    &:focus-visible + ${StyledTextMediaFigure},
+    &[data-garden-focus-visible='true'] ${StyledTextMediaFigure} {
```

Comment:
```suggestion
    &[data-garden-focus-visible='true'] + ${StyledTextMediaFigure} {
```

looks like this changed ... intentionally?

Reply:
Good catch! Will update this.

Reply:
Updated!